docs/eigenlayer/avs-guides/spec/plugin/intro.mdx

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
Don't need to migrate this section. Plugin is a stale concept.

Reply:
Yes - completely forgot! Removing this section again